packages/react-button/src/components/SplitButton/stories/SplitButtonAppearance.stories.tsx

```diff
@@ -2,15 +2,13 @@ import * as React from 'react';
 
 import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-menu';
 
-import { SplitButton, MenuButtonProps } from '../../../index'; // codesandbox-dependency: @fluentui/react-button ^9.0.0-beta
+import { SplitButton, MenuButtonProps } from '../../../index';
 
 export const Appearance = () => (
   <>
     <Menu positioning="below-end">
       <MenuTrigger>
-        {(triggerProps: MenuButtonProps) => (
-          <SplitButton menuButton={triggerProps} primaryActionButton={'This is a split button'} />
-        )}
+        {(triggerProps: MenuButtonProps) => <SplitButton menuButton={triggerProps} primaryActionButton="Default" />}
```

Comment:
The splitbutton menubuttons (here and in the other splitbutton stories) still don't have labels, but LMK if that's intended to be fixed in a separate PR

Reply:
Yeah, I intend to fix those in a separate PR given that that's probably going to be more inside of the component itself to give it a default vs in the stories.